src/styles/index.scss

```diff
@@ -1,3 +1,255 @@
+$stars-size: 16px;
+
 body {
   margin: 0;
+  font-family: Roboto, 'Times New Roman', serif;
+  font-weight: 500;
+}
+
+.container {
+  --column-count: 4;
+
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  justify-content: center;
+  gap: 46px 48px;
+}
```

Comment:
The use of CSS custom properties (variables) like `--column-count` is not specified in the provided checklist. If it is not a part of the project requirements, consider using SCSS variables or a different approach that aligns with the project guidelines.